tests/treatment/checkWhenNoTreatment.test.js

```diff
@@ -26,288 +26,438 @@ const loadObjects = require('../loadObjects');
 // load in all fields with entries prepopulated to null
 const treatment = require('../constructDummyData').getSchemaDummy('treatment');
 
-
 // key -> name of field, value -> unit tests
 const myUnitTests = {
```

Comment:
added a whole lot of tests to cover additional edge cases, including handling extended fields, etc.